ui/plugins/controller.js

```diff
@@ -66,24 +66,29 @@ treeherder.controller('PluginCtrl', [
             var successTab = "jobDetails";
             var failTab = "failureSummary";
 
-            // show/hide Talos panel if this is a Talos job and default to
-            // Talos if the job was a ``success``
-            $scope.tabService.tabs.talos.enabled = (job.job_group_name.indexOf('Talos') !== -1);
-            if ($scope.tabService.tabs.talos.enabled) {
-                successTab = "talos";
-            }
+            // Check if performance data exist
+            $http.get('https://treeherder.mozilla.org/api/project/mozilla-inbound/performance/data/?job_id='+job.id).then(
+                function(response) {
+                    // show/hide Performance panel and default to
+                    // Performance if the job was a ``success``
+                    $scope.tabService.tabs.perfDetails.enabled = !(_.isEmpty(response.data));
+                    if ($scope.tabService.tabs.perfDetails.enabled) {
+                        successTab = "perfDetails";
+                    }
 
-            // Error Classification/autoclassify special handling
-            if ($scope.tabService.tabs.autoClassification.enabled) {
-                failTab = "autoClassification";
-            }
+                    // Error Classification/autoclassify special handling
```

Comment:
These two calls would be better in a ``.then`` after the http call.  That way, if the ``$http.get`` call fails for some reason, we are still able to make the settings based on the other non-perf related values.